samples/GetUserProfile.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+# Get a users display name and avatar
+# Args: host:port username password user_id
+# Error Codes:
+# 2 - Could not find the server.
+# 3 - Bad URL Format.
+# 4 - Bad username/password.
+# 11 - Wrong room format.
+# 12 - Couldn't find room.
+
+
+import sys
+sys.path.insert(0, "../") # add ../ to PYTHONPATH
+
+from matrix_client.client import MatrixClient, User
+from matrix_client.api import MatrixRequestError
+from requests.exceptions import MissingSchema
+from getpass import getpass
+
+host = ""
+username = ""
+room = ""
+
+if len(sys.argv) > 1:
+    host = sys.argv[1]
+else:
+    host = input("Host (ex: http://localhost:8008 ): ")
```

Comment:
Aah, that would explain things, given `input` is a thing in python2, but takes in a python expression, so you need to quote literals.

Is there a nice way to do the Right Thing, do you know?